src/test/java/co/rsk/federate/BtcToRskClientTest.java

```diff
@@ -2398,6 +2399,29 @@ void updateBridgeBtcCoinbaseTransactions_not_removing_from_storage_until_confirm
         verify(coinbases, never()).remove(blockHash);
     }
 
+    @Test
+    void updateBridge_whenFederationIsNull_shouldDoNothing() throws Exception {
+        // Arrange
+        FederatorSupport federatorSupport = mock(FederatorSupport.class);
+
+        BtcToRskClient btcToRskClient = BtcToRskClientBuilder.builder()
+            .withBridgeConstants(BridgeMainNetConstants.getInstance())
+            .withFederation(null)
+            .withFederatorSupport(federatorSupport)
+            .build();
```

Comment:
```suggestion
        PowpegNodeSystemProperties powpegNodeSystemProperties = mock(PowpegNodeSystemProperties.class);
        when(powpegNodeSystemProperties.shouldUpdateBridgeBtcBlockchain()).thenReturn(true);
        when(powpegNodeSystemProperties.shouldUpdateBridgeBtcCoinbaseTransactions()).thenReturn(true);
        when(powpegNodeSystemProperties.shouldUpdateBridgeBtcTransactions()).thenReturn(true);
        when(powpegNodeSystemProperties.shouldUpdateCollections()).thenReturn(true);

        BtcToRskClient btcToRskClient = BtcToRskClientBuilder.builder()
            .withBridgeConstants(BridgeMainNetConstants.getInstance())
            .withFederation(null)
            .withFederatorSupport(federatorSupport)
            .withFedNodeSystemProperties(powpegNodeSystemProperties)
            .build();
```

we should mock this responses to make sure the federator is not calling the methods because of the early return and not because this fields are false by default